README.md

```diff
@@ -57,7 +57,7 @@ This project is based on the work of [Cory Foy](https://github.com/CoryFoy) and
 
```

Comment:
> If you are interested in learning VB.NET or the Full .NET Framework (windows only), please look at his fine work.

I suggest we change this sentence to:

> If you are interested in learning VB.NET, please look at his fine work.

The .NET Framework Koans do not address legacy .NET Framework functionality, as far as I can tell. Additionally, .NET has replaced the old .NET Framework.